src/components/main.js

```diff
@@ -74,7 +74,11 @@ export default class Main extends Component {
 
   render() {
     var component = components[this.props.componentName || "Default"];
-
-    return h(component, this.state);
+    return (
+      <div>
```

Comment:
I'll ask katie how she wants to handle this tomorrow, I believe after some discussion we may change something on this, I'm not sure though. Honestly the issue is that some standard css is going to have to cross components and that master div is not a component. So either we add a class to that div as well styling it, we move the style tag to another area, or we figure out how to return the standard array. There are probably other options but we can discuss it at standup tomorrow